packages/core/src/components/slider/slider.tsx

```diff
@@ -0,0 +1,308 @@
+
+  /**
+   * Minimum slider value
+   */
+  @Prop() min = 0;
+
+  /**
+   * Maximum slider value
+   */
+  @Prop() max = 100;
+
+  /**
+   * Current value of the slider
+   */
+  @Prop() value = 0;
+
+  /**
+   * Define tick marker on the slider. Marker need to be inside min/max
+   */
+  @Prop() marker: SliderMarker;
+
+  /**
+   * Show a trace line
+   */
+  @Prop() trace = false;
+
+  /**
+   * Define the start point of the trace line
+   */
+  @Prop() traceReference = 0;
+
+  /**
+   * Show control as disabled
+   */
+  @Prop() disabled = false;
+
+  /**
+   * Show error state and message
+   */
+  @Prop() error: boolean | string;
+
+  /**
+   *
+   */
+  @Event() valueChange: EventEmitter<number>;
+
+  @State() rangeInput = 0;
+  @State() rangeMin = 0;
+  @State() rangeMax = 100;
+  @State() rangeTraceReference = 0;
+  @State() showTooltip = false;
+
+  private a11yAttributes: A11yAttributes;
+
+  get tooltip() {
+    return this.hostElement.shadowRoot.querySelector('ix-tooltip');
+  }
+
+  get pseudoThumb() {
+    return this.hostElement.shadowRoot.querySelector('.thumb') as HTMLElement;
+  }
+
+  get slider() {
+    return this.hostElement.shadowRoot.getElementById(
+      'slider'
+    ) as HTMLInputElement;
+  }
+
+  @Watch('showTooltip')
+  onShowTooltipChange() {
+    if (this.showTooltip) {
+      this.tooltip.showTooltip(this.pseudoThumb);
+      return;
+    }
+
+    this.tooltip.hideTooltip();
+  }
+
+  componentWillLoad() {
+    this.a11yAttributes = a11yHostAttributes(this.hostElement, [
+      'role',
+      'aria-valuemin',
+      'aria-valuemax',
+      'aria-valuenow',
+    ]);
+    this.updateRangeVariables();
+  }
+
+  @Watch('value')
+  @Watch('max')
+  @Watch('min')
+  @Watch('traceReference')
+  private updateRangeVariables() {
+    if (this.value < this.min) {
+      this.rangeInput = this.min;
+    } else if (this.value > this.max) {
+      this.rangeInput = this.max;
+    } else {
+      this.rangeInput = this.value;
+    }
+
+    this.rangeInput = between(this.min, this.value, this.max);
+    this.rangeTraceReference = between(this.min, this.traceReference, this.max);
+    this.rangeMin = Math.min(this.min, this.max);
+    this.rangeMax = Math.max(this.min, this.max);
+  }
+
+  private onInput(event: InputEvent) {
+    event.stopPropagation();
+    const value = parseInt(this.slider.value);
+
+    if (!isNaN(value)) {
+      this.rangeInput = value;
+      this.emitInputEvent();
+    }
+  }
+
+  private emitInputEvent() {
+    this.valueChange.emit(this.rangeInput);
+  }
+
+  private isMarkerActive(markerValue: number) {
+    const start = Math.min(this.traceReference, this.rangeInput);
+    const end = Math.max(this.traceReference, this.rangeInput);
+    const value = markerValue;
+
+    return value >= start && value <= end;
+  }
+
+  // Listen global on windows because sometimes the event listener
+  // of the dom element input itself is not called if the release
+  // click is not inside the element anymore
```

Comment:
```suggestion
  // Listen globally on window because sometimes the event listener
  // of the DOM element input itself is not called if the release
  // click is not inside the element anymore
```